src/component/rent/RentModalConfirm.tsx

```diff
@@ -53,7 +59,7 @@ const RentModalConfirm = ({
                 ? selectedUser.nickname
                 : selectedUser.email}
             </p>
-            <p className="font-16 color-54">{`현재 대출권수 ( ${selectedUser.lendings.length} / 2 )`}</p>
+            <p className="font-16 color-54">{`현재 대출권수 ( ${selectedUser.lendings.length} / ${selectedUser.role === 2 ? '4' : '2'} )`}</p>
```

Comment:
```suggestion
            <p className="font-16 color-54">{`현재 대출권수 ( ${selectedUser.lendings.length} / ${selectedUser.role >= 2 ? '4' : '2'} )`}</p>
```

DB에는 3번이라고 "기타 운영진"이라는 상태가 하나 더 있어서 사서만 대출해줄게 아니라 기타 운영진도 4권 대출해줄려면 등호가 부등호로 바뀌어야합니다. 물론 지금은 고려 안 해도 되는 문제!

(p.s. 아주 예전에... 사서가 아닌 스태프도 운영에 참가할 수 도 있었던 시설의 레거시입니다.. 현재는 사문화되었지만...)

+

아래에 있는 변수를 enum으로 바꾸어서 사용해서 매직넘버를 없애봐도 좋을거 같네요.
`ex) userRoleStatusEnum["사서"] // 2`

```
// src/constant/status.js
export const userRoleStatus = [
  { code: 0, string: "미인증" },
  { code: 1, string: "카뎃" },
  { code: 2, string: "사서" },
  { code: 3, string: "기타 운영진" },
];
```

Reply:
저도 코드 작성할 때 아쉬웠는데 방법 알려주셔서 감사합니다. 반영 후에 다른 피드백이 있다면 또 알려주세요!